Model/Config/AdditionalConfigVars.php

```diff
@@ -7,6 +7,7 @@ class AdditionalConfigVars implements ConfigProviderInterface
 {
     protected $getInscriptions;
     protected $configProvider;
+    protected $_getInscriptions;
```

Comment:
Are needed 2 attributes with same name? `getInscriptions`  and  `_getInscriptions`

Reply:
already refactored